packages/cli/src/commands/deploy/modules/heroku/templates/scripts/postbuild.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "POSTBUILD DB MIGRATIONS [postbuild.sh]"
+
+node_modules/.bin/rw prisma migrate deploy
+node_modules/.bin/rw prisma generate
```

Comment:
I think so.

Reply:
> Hey @archae0pteryx thanks for your patience. @thedavidprice and I had a chance to review this. We do want to support deploying to heroku, so this PR gets the green light in that sense. But here's some feedback:
> 
>     * I wanted to fix the failing CI check for you but I can't push to this branch because it's from an organization. You should be able to fix it easily though:
>       ```
>       yarn constraints --fix
>       yarn dedupe
>       ```
> 
>     * We're fans of the TUI, but not its screen-clearing behavior. Thoughts on doing away with that part of it?
> 
>     * We want to break this command into 1) `yarn rw setup deploy heroku` and 2) `yarn rw deploy heroku` (setup and deploy commands). That'd better match the existing structure of the other deploy-related CLI commands
> 
>     * Additions to the setup part of the command...
>       
>       * If the heroku CLI isn't found, link to instructions: https://devcenter.heroku.com/articles/heroku-cli#install-the-heroku-cli. Right now the command just spits out an error:
>         ```
>         ╭ 🚀 Heroku Deploy 🚀 ╮
>         │ Starting engines... │
>         ╰─────────────────────╯
>         Command failed with EACCES: heroku apps --json
>         spawn heroku EACCES
>         ```
>       * Make the Route and DB changes optional (via prompts). E.g. “No route at `/` found; would you like to create one?” And “Do you want to add a Heroku Postgres instance?”. In general, the more checks that can be warnings instead of errors the better
> 
>     * This is later down the line, but are you up for adding this to the [deploy target CI repo](https://github.com/redwoodjs/deploy-target-ci)? We'll help of course
> 
>     * This needs docs before it's merged
> 
>     * Lastly, I still need to get feedback from the rest of the core team about the architecture
> 
> 
> Let us know if you have any questions! And feel free to ask me for help with anything.

Sorry for the delay myself. I must have accidentally cleared my notifications at some point and totally missed your comments. @jtoar and @thedavidprice ... This is back in my purview and all of your comments are thumbs up from me.  i will get this forked properly as well so that it can be updated by you guys (among other things mentioned here).

Reply:
Thanks @archae0pteryx We're in to get this to the finish line if you are!

Do know that @jtoar and myself will be out next week. Don't hesitate to nudge us the week after.